doc/devel/coding_style_guideline.rst

```diff
@@ -44,36 +44,36 @@ and readability across the library::
   import numpy.testing as npt
   import scipy as sp
 
-No alias should be used for `h5py`::
+No alias should be used for ``h5py``::
 
   import h5py
 
 -------------------
 Cython coding style
 -------------------
 DIPY recommends the use of the standard Python
-PEP8_ style when writing `Cython <https://cython.org/>` code.
+PEP8_ style when writing ``Cython <https://cython.org/>`` code.
```

Comment:
@areeshatariq This is a link. Simple ticks are OK; however the trailing underscore is missing: it should be
`Cython <https://cython.org/>`_

Reply:
Addressed in ac80972.